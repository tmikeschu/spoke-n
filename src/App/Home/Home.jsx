import React from 'react'
import { Link } from 'react-router'

// Styles
import '../App.css'

const Home = () => (
  <article className="home">
    <img src="http://www.clipartbest.com/cliparts/7ca/6EB/7ca6EBkMi.png" alt="bicycle"/>
    <div className="content">
      <Link to="/landing">Pedal to our site!</Link>
    </div>
  </article>
)

export default Home

