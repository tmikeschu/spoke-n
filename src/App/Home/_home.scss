article.home {
  background: url(../../public/spoken-logo.jpg);
  background-size: 100% 100%;
  background-repeat: no-repeat;
  background-position: 50%;
  min-height: 60em;
  display: flex;
  flex-flow: column nowrap;
  justify-content: flex-end;
  img {
    width: 393px;
    margin-left: 4%;
    margin-bottom: -3%;
  }
  .content {
    background-image: url("https://img.clipartfest.com/d5f409b679ed4ce83abf0fa634cd2d41_png-small-medium-large-large-arrow-clip-art_600-257.png");
    background-repeat: no-repeat;
    background-size: contain;
    width: 100%;
    height: 5em;
    font-size: 2.5em;
    margin-left: 5%;
    margin-bottom: 2%;
    display: flex;
    flex-direction: column;
    justify-content: center;
    a {
      margin-left: 4%;
      width: 30%;
      display: block;
    }
  }

  @media (max-width: 640px) {
    min-height: 30em;
    background-position: 89%;
    background-size: cover;
    justify-content: flex-end;
    img {
      display: none;
    }
    .content{
      background: none;
      margin-bottom: -44%;
      margin-left: 0;
      a {
        text-align: center;
        background-color: #fff;
        padding: 10% 0;
        width: 100%;
        margin: 0;
      }
    }
  }
}
