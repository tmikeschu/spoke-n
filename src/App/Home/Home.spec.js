import React, { Component } from "react"
import Home from "./Home"

describe("<Home />", () => {
  const wrapper = shallow(<Home />)

  it("renders without crashing", () => {
    expect(wrapper).toBeTruthy()
  })

  describe("snapshot", () => {
    it("is valid", () => {
      expect(wrapper).toMatchSnapshot()
    })
  })
})
