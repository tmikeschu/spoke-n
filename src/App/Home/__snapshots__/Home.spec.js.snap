// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> snapshot is valid 1`] = `
<article
  className="home"
>
  <img
    alt="bicycle"
    src="http://www.clipartbest.com/cliparts/7ca/6EB/7ca6EBkMi.png"
  />
  <div
    className="content"
  >
    <Link
      onlyActiveOnIndex={false}
      style={Object {}}
      to="/landing"
    >
      Pedal to our site!
    </Link>
  </div>
</article>
`;
